pkgs/development/python-modules/macropy/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, isPy27
+}:
```

Comment:
How can someone configure the build with the [`extras`](https://github.com/lihaoyi/macropy/blob/46ee500b877d5a32b17391bb8122c09b15a1826a/setup.py#L24)?

Reply:
What is the typical way that this is done?

Reply:
Usually like

```nix
...

pinqSupport ? false, sqlalchemy
}:

propagatedBuildInputs = [
]
++ lib.optional pingSupport sqlalchemy
;
```

##### btw, I don't think the modules for the `extras` are all packaged so you can just add a comment for the others

Reply:
I added pyxl3 and created sqlalchemy and pyxl3 as extras. Fixed.